docs/06-contributors/99-maintainers/050-operations.md

```diff
@@ -20,33 +43,36 @@ The severity level is determined by the impact of the incident on user experienc
 
 | Severity | Description | Example |
 | - | - | - |
-| P0 | Extreme user interruption, no reasonable workaround | Unable to install Wing |
-| P1 | Major degradation of user experience, possible workarounds | Main build is broken in CI (cannot publish) |
-| P2 | The system is usable but has a minor issue. | Error message regression, new message is very confusing |
+| P0 | Extreme/Total user interruption, no reasonable workaround | Unable to install Wing |
+| P1 | Major degradation of user experience, with possible workarounds | Main build is broken in CI (cannot publish) |
+| P2 | Minor degradation of user experience with workaround, or major contributor experience disruption | PR Builds are failing due to misconfiguration |
 
 ## Response Process
 
 ### Acknowledgement
 
-Acknowledgement signifies that someone is taking responsibility for the incident.
-This does not mean that the person acknowledging the incident is who will be resolving it, but they will at least be responsible for triage and communication.
+Acknowledgment signifies that someone is taking ownership over the incident.
```

Comment:
TIL acknowledgment can be spelled two ways 😝

Reply:
Ha I didn't even notice, I put this through Grammarly and suggested the change. tbh I prefer the extra `e`, I'll add it back in.